src/type-helper.d.ts

```diff
@@ -55,6 +55,10 @@ interface Window {
 
 interface Document {
   respecIsReady: Promise<void>;
```

Comment:
I think this returns the config object, so maybe this should be also changed.

Reply:
`respecIsReady` does resolve with config. I was thinking if we should resolve `respec.ready` with nothing. Any use cases to get final config from `ready`?
cc @marcoscaceres

Reply:
I would keep them consistent, tbh. There is no real use case... but we should choose one to deprecate. Having two is confusing... not sure why that happened.

Reply:
[Searched GitHub for `document.respecIsReady`](https://github.com/search?q=document.respecIsReady&type=Code) and found no use of resolved `respecConfig`. So, will be removing it (i.e., `Promise<respecConfig> -> Promise<void>`) in another PR (will still mark it as breaking change).
Will also be deprecating `document.respecIsReady` next, as mentioned in PR description.